test/functional/mempool_truc.py

```diff
@@ -115,7 +112,6 @@ def test_truc_replacement(self):
             from_node=node,
             fee_rate=DEFAULT_FEE,
             utxo_to_spend=utxo_v3_bip125,
-            sequence=MAX_BIP125_RBF_SEQUENCE,
```

Comment:
I’m a bit surprised, why would we have been signaling replaceability for TRUC transactions in the first place? The only nodes that accept V3 transactions would be upgraded to understand TRUC at the same time, so shouldn’t we create TRUC transactions without signaling replaceability?

Reply:
Seems erroneous, indeed.

Reply:
There is a test for both cases v3 + BIP125 and for v3 + non-BIP125 iirc